test/CMakeLists.txt

```diff
@@ -0,0 +1,93 @@
+cmake_policy(SET CMP0135 NEW)
+
+include(FetchContent)
+include(GoogleTest)
+
+set(BUILD_GMOCK OFF)
+set(INSTALL_GTEST OFF)
+set(gtest_force_shared_crt ON)
+
+FetchContent_Declare(
+    gtest
+    URL https://github.com/google/googletest/archive/main.tar.gz
+)
+FetchContent_MakeAvailable(gtest)
+
+add_executable(httplib-test test.cc)
+target_compile_options(httplib-test PRIVATE "$<$<CXX_COMPILER_ID:MSVC>:/utf-8>")
+target_link_libraries(httplib-test PRIVATE httplib GTest::gtest_main)
+gtest_discover_tests(httplib-test)
+
+execute_process(
+    COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_CURRENT_LIST_DIR}/www www
+    COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_CURRENT_LIST_DIR}/www2 www2
+    COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_CURRENT_LIST_DIR}/www3 www3
+    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${CMAKE_CURRENT_LIST_DIR}/ca-bundle.crt ca-bundle.crt
+    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${CMAKE_CURRENT_LIST_DIR}/image.jpg image.jpg
+    WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+    COMMAND_ERROR_IS_FATAL ANY
+)
```

Comment:
Why not use [`file(COPY)`](https://cmake.org/cmake/help/latest/command/file.html#copy)?

Reply:
I forgot the file command at that time. The same thing can be done with less code.

```CMake
file(
    COPY
    ${CMAKE_CURRENT_LIST_DIR}/www
    ${CMAKE_CURRENT_LIST_DIR}/www2
    ${CMAKE_CURRENT_LIST_DIR}/www3
    ${CMAKE_CURRENT_LIST_DIR}/ca-bundle.crt
    ${CMAKE_CURRENT_LIST_DIR}/image.jpg
    DESTINATION ${CMAKE_CURRENT_BINARY_DIR}
)
```

Reply:
Yes, and also note that `${CMAKE_CURRENT_LIST_DIR}/` is unneeded since paths are already relative to the current source directory